### ISSUE TITLE
dark mode not dark in gedit

### ISSUE THREAD

**Issue by Blade04208** [lassekongo83/adw-gtk3#268]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/lassekongo83/adw-gtk3/issues?q=is%3Aissue) to avoid opening a duplicate issue.

### Checklist to make sure the issue is not on your end

- [X] I have made sure that I do not have multiple versions of the theme installed. Only one version should be installed in either `~/.local/share/themes` or `/usr/share/themes` or `~/.themes`
- [X] The file `~/.config/gtk-3.0/gtk.css` does not exist. (If it does, backup and remove it.)
- [X] I have logged out and in again.
- [X] Dark theme only. I have enabled adw-gtk3-dark in `gnome-tweaks` AND set the appearance to dark in `gnome-control-center`
- [X] The flatpak theme is installed `org.gtk.Gtk3theme.adw-gtk3` and `org.gtk.Gtk3theme.adw-gtk3-dark`
- [X] The app I'm having an issue with is an actual GTK3 app and is not coded using another toolkit like GTK2, GTK4, QT, etc.
- [ ] I have created a new user profile without any modifications except adw-gtk3.

### Description

I dont know if this is just because gedit wants to be light, but gedit will only stick to light mode. Uses the theme just fine, just dosent use dark mode.
![image](https://github.com/user-attachments/assets/9ad6c6ca-8555-48dd-bccf-7e8dcf9d0b13)
`note: i am a dumbass, i do dumbass things, take everything i say with a bit of patience.`

### OS and version [e.g Fedora 37]

EndeavourOS Endeavour (great name, i know)

### Adw-gtk3 version. Also specify your installation source. [e.g tar.xz, AUR, git main branch]

5.3.2 - AUR

### GNOME version

46.4

### Application name and version (If the issue is with an app.)

gedit 47.0

**Comment by lassekongo83** [comment]
> AND set the appearance to dark in `gnome-control-center`

Did you actually do this?

**Comment by Blade04208** [comment]
yes, i did actually do that, its the first thing i do every time i install gnome
although this could be the thing causing it... 
![image](https://github.com/user-attachments/assets/12776731-b4d4-4a7f-888d-34d7963adcff)
ill try setting that to 1 and restart my session now, and close this issue if that is what was causing it.

**Comment by Blade04208** [comment]
> Did you actually do this?

yes, i did actually do that, its the first thing i do every time i install gnome

although this could be the thing causing it... 
![image](https://github.com/user-attachments/assets/12776731-b4d4-4a7f-888d-34d7963adcff)
setting this to 1 (and restarting my session, ofc) did not do anything to gedit.

**Comment by lufog** [comment]
@Blade04208, this may be a strange question, but are you sure you have `Follow system style` selected?

![image](https://github.com/user-attachments/assets/bba25997-d695-45c3-a3ab-7d25e1cbc9ee)
